packages/libs/wdk-client/src/Components/Display/CollapsibleDetailsSection.tsx

```diff
@@ -1,17 +1,24 @@
-import React, { ReactNode, useState } from 'react';
+import React, { ReactNode, useEffect, useState } from 'react';
 
 interface Props {
   summary: ReactNode;
   collapsibleDetails: ReactNode;
   initialShowDetailsState?: boolean;
+  expandDueToFiltering?: boolean;
```

Comment:
To be honest, I'm not crazy about this new prop. It might be time to consider making this a fully controlled component, and remove (e.g., "lift") the `showDetails` state. I'm not sure how much work this would entail. (How many different places would need to be updated to track this state?)

Reply:
Yeah, great point. I'll push a change shortly. Shouldn't be a big load given that it's a new component being used in only 4 instances: two sections in clinepi studies menu, two in mbio.